Jenkinsfile

```diff
@@ -1,7 +1,11 @@
-// Builds a module using https://github.com/jenkins-infra/pipeline-library
-buildPlugin(useContainerAgent: true, configurations: [
-    [platform: 'linux', jdk: 11], 
-    [platform: 'windows', jdk: 11],
-    [platform: 'linux', jdk: 17],
-    [platform: 'linux', jdk: 21]
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/
+buildPlugin(
+  forkCount: '1C', // run this number of tests in parallel for faster feedback.  If the number terminates with a 'C', the value will be multiplied by the number of available CPU cores
+  useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
+  configurations: [
+    [platform: 'linux', jdk: 21],
```

Comment:
Should we align with https://github.com/jenkinsci/jenkins/blob/master/Jenkinsfile#L16 ?